zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -1272,6 +1272,44 @@ paths:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
 
+  /elements/{elementKey}/variabels:
+    post:
+      tags:
+        - Variables
```

```suggestion
        - Elements
```

🔧 is this endpoint about the "Variables" resource, or about the "Elements" resource? To me it sounds like Elements, and you're setting Variables on an Element.

---

Yes, it's for element resources. We can use both tags, wdyt ?

---

For each tag, we add a navigation category in our reference docs. The endpoint is added to all categories then. This is a bit weird, I guess, so we should focus on one tag, I think. Voting for "Element instances" since this is the resource we're working on.

---

See [here](https://github.com/camunda/camunda-docs/pull/4217#discussion_r1742669720) what happens with multiple tags

---

@pepopowitz, just noticed, we currently use the singular form for the tags, not the plural. This is (mostly) what Operate, Optimize, and Tasklist did as well in their REST APIs so far.

---

I'd go for element instance